policy-test/tests/api.rs

```diff
@@ -331,3 +319,20 @@ fn mk_admin_server(ns: &str, name: &str) -> k8s::policy::Server {
         },
     }
 }
+
+async fn retry_watch_server(
+    client: &kube::Client,
+    ns: &str,
+    pod_name: &str,
+) -> tonic::Streaming<grpc::inbound::Server> {
+    // Port-forward to the control plane and start watching the pod's admin
+    // server's policy and ensure that the first update uses the default
+    // policy.
+    let mut policy_api = grpc::PolicyClient::port_forwarded(&client).await;
```

Comment:
clippy flags this as a needless borrow

```suggestion
    let mut policy_api = grpc::PolicyClient::port_forwarded(client).await;
```

Reply:
TIL about the undocumented `--tests` flag for `cargo clippy`

Reply:
I tend to use `--all-targets`. I'll get the justfile from my branch cleaned up in a separate review.